### ISSUE TITLE
Couldn't load custom interfaces

### ISSUE THREAD

**Issue by mattiaz9** [directus/directus#6526]
### What version of Directus are you using?

9.0.0-rc.81

### What version of Node.js are you using?

14.13.0

### What database are you using?

Postgres

### What browser are you using?

Safari, Chrome

### What operating system are you using?

macOS

### How are you deploying Directus?

running locally

### Describe the Bug

I'm trying to migrate my interfaces to the latest specifications. I followed the guide here https://docs.directus.io/guides/interfaces/ but still I can't load my interfaces.

After editing the directus code I was able to print the error which is: `TypeError: Failed to fetch dynamically imported module: http://localhost:8055/extensions/interfaces/index.js`

I'm not sure what I'm doing wrong.

### To Reproduce

This is my rollup config:
```
import { terser } from 'rollup-plugin-terser'
import commonjs from '@rollup/plugin-commonjs'
import { nodeResolve } from '@rollup/plugin-node-resolve'
import vue from 'rollup-plugin-vue'

const dev = process.env.ROLLUP_WATCH === "true"

const plugins = [
  vue({ preprocessStyles: true }),
  nodeResolve(),
  commonjs(),
  !dev && terser(),
]

const external = [
  'vue',
  '@directus/extension-sdk'
]

export default [{
  input: 'src/interfaces/my-interface/index.js',
  output: { file: 'extensions/interfaces/my-interface/index.js', format: 'es' },
  external, plugins,
}]
```

This is the vue file:
```
<template>
  <div>Hello</div>
</template>


<script>
export default {
}
</script>
```

**Comment by nickrum** [comment]
I can't reproduce this on my end. Your extension is working perfectly fine.

What's strange is that `http://localhost:8055/extensions/interfaces/index.js` should never fail even if there aren't any extensions loaded. Do you get the same error if you aren't using any extensions? Could you try to open that URL directly inside a browser and see if it produces any errors?
Also, does the output from the API show any errors?

**Comment by mattiaz9** [comment]
Opening the url directly doesn't produce any error, but I noticed this import at the first line:
`import { openBlock, createBlock } from './admin/vue.089cc4dd.js';`

... and in the browser, before the error I got this 404: `http://localhost:8055/extensions/interfaces/admin/vue.089cc4dd.js`

**Comment by nickrum** [comment]
Is your `PUBLIC_URL` set to `/`? This will be fixed by #6534.

**Comment by mattiaz9** [comment]
If you refer to the one in the `.env`, yes is set to `/`
